static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -19,24 +19,24 @@ import fastdom from 'fastdom';
 import raven from 'lib/raven';
 import userPrefs from 'common/modules/user-prefs';
 import images from 'common/modules/ui/images';
-import storage from 'lib/storage';
+import { localStorage } from 'lib/storage';
```

Comment:
Agree with your suggestion. Although you have to go all the way back to IE7 before web storage becomes an issue! Perhaps it's finally time to kill this shim